master_changes.md

```diff
@@ -81,6 +81,10 @@ New option/command/subcommand are prefixed with ◈.
   * Restrict to opam-file-format 2.1.2 [#4495 @rjbou]
   * Switch to newer version of MCCS (based on newer GLPK) for src_ext [#4559 @AltGr]
   * Bump dune version to 2.8.2 [#4592 @AltGr]
+  * Update opam file to 2.0 [#4371 @AltGr]
+  * Makefile: Add rule `custom-libinstall` for `opam-custom-install` use [#4401 @AltGr]
+  * opam-client lib: fix dependency constraints for cmdliner, extlib [#4410 @AltGr]
+  * 4.12 compatibility [#4437 @dra27]
```

Comment:
Some of changes are already in beta4 one

Reply:
Fixed